prebuilt-tasks/pom.xml

```diff
@@ -77,5 +77,17 @@
             <groupId>org.assertj</groupId>
             <artifactId>assertj-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>dev.parodos</groupId>
+            <artifactId>notification-service-sdk</artifactId>
+            <version>${revision}</version>
+            <scope>compile</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-web</artifactId>
+            <version>5.3.26</version>
+            <scope>compile</scope>
```

Comment:
same about scope here

Reply:
I think you can also remove `<version>5.3.26</version>` because all the spring deps are imported now in the parent. Can you remove it and check please?

Reply:
Both removed